### PR TITLE
Ensure Rails benchmarks can run with errors.

### PR DESCRIPTION
**Pull request by tgxworld**
@kirs  Could you review this? Thanks!

### CODE REVIEW

File: rails/benchmarks/bm_activemodel_views.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
 require 'rails'
 require 'action_controller/railtie'
 require 'active_model'
+require 'ffaker'
```
Comment (kirs) [inline rails/benchmarks/bm_activemodel_views.rb]: As I remember, we decided to remove `ffaker` from benchmarks

File: rails/benchmarks/bm_app.rb [hunk 2]
```diff
@@ -1,29 +1,321 @@
+require 'bundler/setup'
+require 'rails'
+require 'action_controller/railtie'
+require 'active_record'
+require 'ffaker'
```
Comment (kirs) [inline rails/benchmarks/bm_app.rb]: Drop ffaker from here too?

File: rails/benchmarks/driver.rb [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
 require 'optparse'
+require 'rails'
 
 RAW_URL = 'https://raw.githubusercontent.com/ruby-bench/ruby-bench-suite/master/rails/benchmarks/'
 
+sqlite3_url =
+  if Rails.version < '4.1.0'
+    'sqlite3:/:memory:'
+  else
+    'sqlite3::memory:'
```
Comment (kirs) [inline rails/benchmarks/driver.rb:17]: :grimacing: does it solve the problem you experienced with failing connection?

**Comment by kirs** [comment]
Good job!

File: rails/benchmarks/bm_activemodel_views.rb (hunk 1 above)
Comment (tgxworld) [inline rails/benchmarks/bm_activemodel_views.rb]: Ah yes. I'll update the benchmark with `Lorem Ipsum`

File: rails/benchmarks/driver.rb (hunk 3 above)
Comment (tgxworld) [inline rails/benchmarks/driver.rb:17]: yup it does now. For some reason I couldn't get the full url to work.

Comment (kirs) [inline rails/benchmarks/driver.rb:17]: :+1:

**Comment by tgxworld** [comment]
@kirs Thanks for reviewing! I couldn't get `bm_app.rb` and `bm_activemodel_views.rb` to work on Rails 3.2.0 yet so I disabled them for now.

**Comment by kirs** [comment]
:rocket:
Can I help with integrating it into rubybench.org?

**Comment by tgxworld** [comment]
Sure! I'm actually working on the Docker container right now.

**Pull request merged**